presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java

```diff
@@ -68,7 +68,7 @@ public static JdbcClient createJdbcClientWithStats(JdbcClient client)
     {
         StatisticsAwareJdbcClient statisticsAwareJdbcClient = new StatisticsAwareJdbcClient(client);
 
-        Logger logger = Logger.get(JdbcClient.class);
+        Logger logger = Logger.get(client.getClass());
```

Comment:
Hm. This makes it rather not explicit. it may be eg "MySqlClient" or it can be "DoingSomethingWrappingClient" or even "MySqlClientModule$1" (as in the case of `new ForwardingJdbcClient() {..}` created at the end of this method).

I see the value of changing this -- distinguishing between calls in different connectors. Say, i want to log in mysql connector but not in postgres. I propose doing this differently though, or holding off with this change for now.

Reply:
Maybe I could setup a logging for `io.prestosql.plugin.jdbc.${catalog_name}" instead?

Reply:
Good idea.
Maybe `io.prestosql.plugin.jdbc.${catalog_name}.jdbcclient`? (i lowercased `JdbcClient` to signal this is not a class name; i retained it to make this more contextual)